sdk/pnpm-workspace.yaml

```
@@ -1,3 +1,15 @@
 packages:
   - integration-tests
+  - examples/*
   - packages/*
+
+catalog:
+  "@tanstack/react-query": "^5.69.0"
+  "@testing-library/dom": "^10.4.0"
+  "@testing-library/react": "^16.2.0"
+  "@types/react": "^18.3.1"
+  "@types/react-dom": "^18.3.0"
+  "wagmi": "2.15.1"
+  "viem": "2.28.1"
```

Should use `^` to ensure any v2.x versions are supported
```suggestion
  "wagmi": "^2.15.1"
  "viem": "^2.28.1"
```

---

will do